ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master__4.11.yaml

```
@@ -121,9 +121,11 @@ tests:
         export TEST_PERFORMANCE_USERS_PER_SCENARIO=400
         export REGISTRY_NAMESPACE=ocpappsvc-qe
         source /var/run/secrets/sbo-on-sb-perf/registry-credentials-env.sh
+        source /var/run/secrets/sbo-on-sb-perf/opensearch-aws-credentials-env.sh
         make -k test-performance test-performance-collect-kpi || true
         make test-performance-artifacts
-        make test-performance-thresholds
+        make test-performance-thresholds || true
+        make test-performance-upload-kpi
```

```suggestion
        make test-performance-upload-kpi || true
        make test-performance-thresholds
```
This step is supposed to fail the job if the thresholds are exceeded and so it should be the last step in the job and without the `|| true`.

---

changed the sequence now in the latest commit